### PR TITLE
Small usability changes

### PR DESCRIPTION
**Pull request by soraphis**
- made amounts in MoneyUnit more readable, by adding thousands separator. 
- added virtual size to WriteTransaction for debugging reasons when using SegWit

edit: see changes here https://github.com/MetacoSA/NBitcoin/pull/822/files?utf8=%E2%9C%93&diff=split&w=1
(hide whitespace. should think about adding a .gitattributes to disable gits autocrlf conversion)

### CODE REVIEW

File: NBitcoin/RPC/BlockExplorerFormatter.cs [hunk 1]
```diff
@@ -1,126 +1,127 @@
...
+			WritePropertyValue(writer, "hash", tx.GetHash().ToString());
+			WritePropertyValue(writer, "ver", tx.Version);
+
+			WritePropertyValue(writer, "vin_sz", tx.Inputs.Count);
+			WritePropertyValue(writer, "vout_sz", tx.Outputs.Count);
+
+			WritePropertyValue(writer, "lock_time", tx.LockTime.Value);
+
+			WritePropertyValue(writer, "size", tx.GetSerializedSize());
+			WritePropertyValue(writer, "virtual_size", tx.GetVirtualSize());
```
Comment (nopara73) [inline NBitcoin/RPC/BlockExplorerFormatter.cs]: Could be a backwards compatibility issue.

File: NBitcoin/Money.cs [hunk 2]
```diff
@@ -1,904 +1,904 @@
...
+			if (id is null)
+				return new AssetMoney(assetId);
+			return new AssetMoney(id, result);
+		}
+	}
+
+	public enum MoneyUnit : int
+	{
+		BTC = 100_000_000,
+		MilliBTC = 100_000,
```
Comment (nopara73) [inline NBitcoin/Money.cs]: ACK

**Comment by nopara73** [comment]
Can you fix your whitespace issues in the future? (@NicolasDorier may asks for fixing it here too.) The no-whitespace mode sometimes cheats the reviewers into thinking something didn't happen when it did. It'd be a way to add malicious code to a library.

**Comment by knocte** [comment]
Three changes in one same commit? I'd recommend separating at least into 3 commits (if not 3 PRs).

**Comment by soraphis** [comment]
Okay, i'll address those points. may take a while

**Pull request closed without merging**